src/api/app/views/webui2/webui/main/_breadcrumb_items.html.haml

```diff
@@ -5,5 +5,5 @@
     = home_title
```

Comment:
It's somewhat weird now, because on the homepage, this is what we see:

![homepage](https://user-images.githubusercontent.com/1102934/49881200-13446400-fe2e-11e8-92a3-a032c0cb1fa0.png)

And on the other pages:
![project-show](https://user-images.githubusercontent.com/1102934/49881192-0fb0dd00-fe2e-11e8-9f96-92da1ff85b86.png)

I think `home_title` should also be removed here. There should also be a title as I said in my other comment.

Reply:
I added some reasoning in my other comment. I wanted to get some feedback first before changing this further.